pkg/pool-weighted/contracts/smart/LiquidityBootstrappingPool.sol

```diff
@@ -45,7 +70,8 @@ contract LiquidityBootstrappingPool is BaseWeightedPool, ReentrancyGuard {
         uint256 swapFeePercentage,
         uint256 pauseWindowDuration,
         uint256 bufferPeriodDuration,
-        address owner
+        address owner,
```

Comment:
I would add a requirement that this isn't the zero address (potentially not event delegated ownership?). It would prevent headaches from someone shelling out $ on a useless pool.

Reply:
Maybe not on disallowing delegated ownership. That's something that's much harder to do by accident.

Reply:
The CRP did this (prevented transferring control to 0 - it was initially set to the sender, which can't be zero, so it didn't have the same initialization issue).

Of course, you can always defeat it by setting the owner to a non-zero burn address, but that would be deliberate. An error in a script could cause the owner to be set to zero accidentally. It's not much gas, and only on construction.